CONTRIBUTING.md

```diff
@@ -75,16 +75,75 @@ $ go test -v -short ./...
 
 ## Releasing
 
-Releases are automatically prepared using GitHub actions [here](.github/workflows/release.yml).
-In general terms, whenever a tag with `v*` pattern is pushed up, the following steps
-are automatically run:
-- Tests are re-run
-- If tests fail, release process is halted
-- Release notes and changelog files are generated
-- Draft release is created on GitHub
-- Release notes and the changelog are attached as assets to that release
-
-To view the progress of the actions, you can take a look at [this](https://github.com/cyberark/conjur-oss-suite-release/actions) page.
-
-After these steps run and they encounter no problems, manual publishing of release will make it
-public for consumers of this repo.
+1. Determine whether there are component changes since the last suite release
+   that merit a new suite release.
+   
+   - Check the [wiki](https://github.com/cyberark/conjur-oss-suite-release/wiki/Unreleased-Changes)
+     to see the daily report on which components have had new tagged
+     versions since the last release, and which components have unreleased changes.
+     - Note: Entries for components with unreleased changes (changes on the master branch that are not
+       yet available in a GitHub release) show in this report as `org/repo @HEAD`. The links take you
+       to the commit history for all commits on master that are not included in the latest GitHub release.
+       
+   - If there are any components with unreleased changes that should be tagged, open an
+     issue in that component's repository for adding a new tag.
```

Comment:
Maybe also tag the appropriate group (core/community)?

Reply:
I actually added an issue today to add us as CODEOWNERS to all of our repos. I'd like to propose that the core team does this as well, and then this will not be required

Reply:
Makes sense though it probably shouldn't be fully assumed that the file is there

Reply:
I think I'd like to leave it without that. If our processes aren't set up so that we're notified properly when someone opens an issue in one of our repos, then that's on us - I'm not going to add a step to issue reporters' workflows to make sure we pay appropriate attention to our stuff